crates/wasm-compose/src/encoding.rs

```diff
@@ -1083,7 +1083,7 @@ impl<'a> ImportMap<'a> {
             assert!(self
                 .0
                 .insert(
-                    entry.component.name.to_kebab_case().into(),
+                    entry.component.name.clone().into(),
```

Comment:
I think we might be able to lose the `Cow<'a, str>` key for the import map and just have `&'a str` now that we're no longer needing to create a new string by `to_kebab_case()`.